src/System.Management.Automation/engine/PSVersionInfo.cs

```diff
@@ -42,6 +41,13 @@ public class PSVersionInfo
 
         private static readonly PSVersionHashTable s_psVersionTable;
 
+        /*
```

Comment:
Use of multiline comment seems a bit out-of-place.  Consider using `//`

Reply:
Using `//` raise CodeFactor issue (blank line after the comment).

Reply:
I think CodeFactor recommends using `////` to comment out code specifically, which bypasses the usual warning for that.

Reply:
@xtqqczze Have you thoughts?